### DISCUSSION TITLE
Replace 4:3 black bars with custom bezel

### DISCUSSION THREAD

**Discussion by Tenome** [emoose/OutRun2006Tweaks#215] [Suggestions]
### Describe the feature you'd like to see added

Lambada brought up the idea in the discord, so I'm reposting it here. I'm guessing this would be done by drawing the texture over the game, but would that cause issues with alt-tabbing or losing window focus? Is there a better way?

![sidebars-4x3-preview-03-outrun2sp](https://github.com/user-attachments/assets/b342b70a-3456-4612-a786-74b75f0b9364)

**Comment by Zoukambada** [comment]
Thank you @Tenome for creating this issue here.

I all attach here the bezels that I have created so far. This way this feature can be beta tested with it.

The first bezel is in 4K:
![sidebars-4x3-4k-bezel](https://github.com/user-attachments/assets/6a3991e0-1a07-4b3e-847e-36ddbff3b07a)

The second bezel is in 1080p:
![sidebars-4x3-1080p-bezel](https://github.com/user-attachments/assets/2194d537-4a9d-499a-9858-dbae6d6599a1)

I also created two more variants that are transparent. This way this feature could be very customizable, for example:
_Add a transparent bezel and select the color behind it, if set the HEX RGB code for RED #d32829 the color would affect the transparent bezel._

Example:
![sidebars-4x3-preview-05-customcolor](https://github.com/user-attachments/assets/48513079-df06-4805-9f32-055ff6db14d7)

Bezel in 4k but transparent:
![sidebars-4x3-4k-bezel-transparent](https://github.com/user-attachments/assets/24c82a2d-40d4-43bb-8d5d-337a6471bda7)

Bezel in 1080p but transparent:
![sidebars-4x3-1080p-bezel-transparent](https://github.com/user-attachments/assets/f67c0a76-1bb6-445c-8a6d-a55754314478)

**Comment by Zoukambada** [comment]
Another sample applied to the intro movie:

![sidebars-4x3-preview-06-goldenintro](https://github.com/user-attachments/assets/62bccdb7-5983-4234-b06c-9a50b7b607dd)

**Comment by samgrant78** [comment]
I love the red one.
